src/miner/miner.rs

```diff
@@ -277,6 +277,7 @@ impl CuckooMinerJobHandle {
 			if r.has_stopped {
 				break;
 			}
+			thread::sleep(time::Duration::from_millis(1));
```

Comment:
And thanks for looking into the busy waits here, I know there are a few within cuckoo miner I just haven't had time to chase them down yet.